hack/build-c.sh

```diff
@@ -12,6 +12,23 @@ export GOBIN=${PWD}/bin
 export GOPATH=${PWD}/gopath
 export CGO_ENABLED=1
 
+GIT_COMMIT=$(git log -1 --pretty=format:%h)
+GO_COMMIT_VAR="github.com/openshift/app-netutil/lib/v1alpha.GitCommit"
+
+GIT_VERSION=$(git tag | sort -V | tail -1)
```

Comment:
nit: GIT_VERSION currently has no value, is it worth using the default value "development" set in lib/v1alpha?

Reply:
@zshi-redhat I was thinking we should add a "v1.0.0" soon. I have a few more changes this week then add the tag. If we want to test it, we could add a "v0.5.0" tag to master now, but don't think it is needed.

Reply:
agree, no need to add a tag only for testing, we can wait for v1.0.0.